content/en/blog/_posts/2022-08-04-cgroupv2-ga.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: blog
+title: "Kubernetes 1.25: cgroup v2 graduates to GA"
+date: 2022-08-31
+slug: cgroupv2-ga-1-25
+---
+
+**Authors:**: David Porter (Google), Mrunal Patel (RedHat)
+
+Kubernetes 1.25 brings cgroup v2 to GA (general availability), letting the
+kubelet use the latest resource management capabilities.
+
+## What are cgroups?
+
+Effective [resource management](/docs/concepts/configuration/manage-resources-containers/) is a
+critical aspect of Kubernetes. This involves managing the finite resources in
+your nodes, such as CPU, memory, and storage.
+
+*cgroups* are a Linux kernel capability that establish resource management
+functionality like limiting CPU usage or setting memory limits for running
+processes.
+
+When you use the resource management capabilities in Kubernetes, such as configuring
+[requests and limits for Pods and containers](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits),
+Kubernetes uses cgroups to enforce your resource requests and limits.
+
+The Linux kernel offers two versions of cgroups: cgroup v1 and cgroup v2.
+
+## What is cgroup v2?
+
+cgroup v2 is the latest version of the Linux cgroup API. cgroup v2 provides a
+unified control system with enhanced resource management capabilities.
+
+cgroup v2 has been development in the Linux Kernel since 2016 and in recent
+years has matured across the container ecosystem. With Kubernetes 1.25, cgroup
+v2 support is graduating to GA.
+
+Many new Linux distributions have switched over to cgroup v2 by default so it's
```

Comment:
```suggestion
Many Linux distributions have switched over to cgroup v2 by default so it's
```

Also not-so-new distributions such as Ubuntu and, er, Debian (started in 1993). I'd drop the “new”, or clarify that you mean “recent releases of”.

Reply:
good point, reworded to "recent releases of"